EdFi.Ods.Utilities.Migration/Scripts/02 Upgrade/v34_to_v50/01 Structure/1200-AddPrimaryKeysForV50.sql

```diff
@@ -0,0 +1,5 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
```

Comment:
I meant to comment on this one last round, but if there is nothing in this script, and the code doesn't require it for some reason, than I don't see any reason to keep it.

Reply:
I removed it . Thanks